app/views/projects/blob/_replace.html.haml

```diff
@@ -0,0 +1,24 @@
+#modal-replace-blob.modal
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %a.close{href: "#", "data-dismiss" => "modal"} ×
+        %h3.page-title Replace #{@blob.name}
+        %p.light
+          From branch
+          %strong= @ref
+      .modal-body
+        = form_tag namespace_project_blob_path(@project.namespace, @project, @id), method: :put, class: 'form-horizontal', :multipart => true  do
+          %br
+          .form-group
+            .col-sm-offset-2.col-sm-10
+              = file_field_tag :file_upload, :id => "file_upload", :class => "file_up", :required => "required"
+          %br
+          = render 'shared/commit_message_replace_container', params: params,
+                  placeholder: 'Replace this file because...'
+          .form-group
+            .col-sm-offset-2.col-sm-10
+              = button_tag 'Replace file', class: 'btn btn-small btn-primary btn-remove-file', id: 'replace_btn'
+              = link_to "Cancel", '#', class: "btn btn-cancel", "data-dismiss" => "modal"
+:coffeescript
+  disableButtonIfAnyEmptyField $('#commit_message_replace'), $('.btn-remove-file')
```

Comment:
These calls have been removed in favor of a new behavior.

See https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/900